cleanup_utils/integration_cleanup.py

```diff
@@ -77,16 +77,16 @@ def run_integration_cleanup(dry_run=False):
             "resources."
         )
 
-    # Clean up AWS
+    # Do a dry-run cleanup to get the hashes.
```

Comment:
@lgp171188 Why run a dry run of everything before running them again? You can just run the other cleanups first and move this to the end.